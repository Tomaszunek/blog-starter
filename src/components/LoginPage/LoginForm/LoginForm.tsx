import * as React from 'react';

export default class LoginForm extends React.Component<any, any> {    
  public render() {
    return (
      <div className="loginForm">
        <form action="">
          <span>Username</span>
          <input type="username"/>
          <span>Password</span>          
          <input type="password"/>
          <button className="sign-in"  type="submit" >Sign in</button>
          <a href="">Forgot password?</a>
        </form>      
      </div>
    );
  }
}


