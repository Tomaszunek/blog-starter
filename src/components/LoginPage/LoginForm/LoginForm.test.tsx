import * as React from 'react';
// import { mount } from 'enzyme';
import * as renderer from 'react-test-renderer';
import LoginForm from './LoginForm'

describe('LoginForm Comp', () => {
  describe('Component should render right', () => {    
    it.skip('render correctly snapshot component', () => {
      const tree = renderer
      .create(<LoginForm/>)
      .toJSON();
      expect(tree).toMatchSnapshot();
    })    
  });
});