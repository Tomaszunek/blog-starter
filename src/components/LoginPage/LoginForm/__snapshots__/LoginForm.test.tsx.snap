// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm Comp Component should render right render correctly snapshot component 1`] = `
<div
  className="loginForm"
>
  <form
    action=""
  >
    <span>
      Username
    </span>
    <input
      type="text"
    />
    <span>
      Password
    </span>
    <input
      type="password"
    />
    <button
      className="sign-in"
      type="submit"
    >
      Sign in
    </button>
    <a
      href=""
    >
      Forgot password?
    </a>
  </form>
</div>
`;
